### ISSUE TITLE
Generated files idempotence

### ISSUE THREAD

**Issue by gf-news** [go-jet/jet#33]
Thank you for your work on this library - a very nice design idea and very slick in use!

I'm driving the `jet` executable from go generate, and it works very well, except jet always adds the generation date which ends up being an only thing in the git diff after the generation step and requires me to then revert the jet generated files that are not actually changed.

```
//
// Code generated by go-jet DO NOT EDIT.
// Generated at Wednesday, 15-Jan-20 08:38:53 GMT
//
```

Would it be possible to remove or introduce a config switch for the date generation?

Thanks and well done on the library again!

**Comment by go-jet** [comment]
Hi @gf-news, thanks for reporting the issue.
You've made a valid point. File generation should be idempotent.

**Comment by go-jet** [comment]
@gf-news Timestamp from generated files has been removed on `idempotency-gen` branch(_the branch is in sync with latest release_). You can use this branch until the next official release.

**Comment by gf-news** [comment]
Thanks a lot!

**Comment by go-jet** [comment]
This change is now included in latest v2.3.0 release.
